cypress/integration/Select/position.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Position of select menu dropdown
```

Comment:
We're not exposing the select. Currently this is only a helper to reduce shared code between the `SingleSelect` and the `MultiSelect`. I think we should have two (probably identical) feature files for those two, then the underlying architecture doesn't matter